### ISSUE TITLE
Configuration | Settings generates Error 500: Internal Server Error

### ISSUE THREAD

**Issue by khorjames** [openboxes/openboxes#2458]
URI: /openboxes/admin/showSettings
Class: java.lang.NullPointerException
Message: Request processing failed; nested exception is org.grails.gsp.GroovyPagesException: Error processing GroovyPageView: null
Caused by: null
<img width="960" alt="Screenshot 2021-06-08 at 11 31 36 PM" src="https://user-images.githubusercontent.com/31814643/121214438-b49eba80-c8b1-11eb-9410-1fd51ec0903c.png">

**Comment by jmiranda** [comment]
Yeah there are lots of bugs with the Grails 3 version and we're working our way through that backlog. 

We have an initial test script here 
https://docs.google.com/spreadsheets/d/1G4rOQzn7fIj9sGgUytPxhdVT19x8Ahbyj64xhN0ph-I/edit#gid=0

And the bugs / improvements are being tracked in a private Jira project. We can probably make that available to external users if you're interested in contributing. Otherwise continue to add these bugs here and we'll consolidate at a later time.

**Workaround**
The bug you found above has a workaround. Just set the Locale on your user and this error will go away.

**Comment by khorjames** [comment]
Hi Justin,

I had tried to set localisation for my user as seen in the screenshot below. Unfortunately, I cannot add localisation.





Thank you.

Best Regards,
James

**Comment by jmiranda** [comment]
Sorry I should have been more explicit. You can set the default locale on the user through the Edit User page. 
![image](https://user-images.githubusercontent.com/1162935/121625923-0e31f100-ca3a-11eb-830b-35bf513abcb2.png)

**Comment by khorjames** [comment]
Hi Justin,

Sorry to trouble you again.

I logged in as admin but it does not have the privilege to edit user.



I created a new user called "jameskhor” but I don’t have the privilege to enable it.

Thanks.

**Comment by jmiranda** [comment]
No worries. You should be able to login as "admin" with password "password". And then edit your "jameskhor" user to add a role like ROLE_SUPERUSER or ROLE_ADMIN.
